test/src/main/java/org/eclipse/kuksa/test/kotest/Tag.kt

```diff
@@ -22,7 +22,10 @@ package org.eclipse.kuksa.test.kotest
 import io.kotest.core.NamedTag
 
 val Integration = NamedTag("Integration")
+val DefaultDatabroker = NamedTag("DefaultDatabroker") // unsecure => no tls, no authentication
+val CustomDatabroker = NamedTag("CustomDatabroker")
+val Secure = NamedTag("Secure") // tls enabled, no authentication
+val Authentication = NamedTag("Authentication") // no tls, authentication enabled
```

Comment:
recommendation to execute all normal tests (Unit, Integration w. DefaultDatabroker): 
./gradlew clean build -Dkotest.tags="!CustomDatabroker"

we should think about removing Integration because Integration can only be used paired with Secure, Insecure or Authentication so not a really useful category anymore.

Reply:
Decided to leave it inside so it can be used for the negation: -Dkotest.tags="!Integration" to execute all non-Integration tests. 
Basically we do the same with CustomDatabroker tag. It self has no purpose because CustomDatabroker can either be authentication or secure / tls. However when used with the negation we can execute all tests which don't use a "custom databroker"